### PR TITLE
upgrade all dependencies, mainly to support react-redux 7 and react-router 5

### PR DESCRIPTION
**Pull request by badtant**
Since I requested it here https://github.com/makeomatic/redux-connect/issues/139
I though I might pull it of myself :)

### CODE REVIEW

**Comment by AVVS** [comment]
thanks @badtant 

CC @aleksxor could you review the code and make changes if need be / merge to support updates"

File: package.json [hunk 1]
```diff
@@ -39,49 +39,49 @@
   },
   "homepage": "https://github.com/makeomatic/redux-connect",
   "peerDependencies": {
-    "prop-types": "16.x.x",
+    "prop-types": "15.x.x",
     "react": "16.x.x",
```
Comment (aleksxor) [inline package.json]: i'd assume to tighten react peer dep for a bit as react-redux@7 requires ^16.8.4

Comment (adylevy) [reply package.json]: Agree, I'd go with latest react if possible

**Comment by badtant** [comment]
so, are you ready to merge? :)

**Comment by AVVS** [comment]
yes, if you can change react peer dep to `^16.8.4` then I'd be happy to merge

**Comment by badtant** [comment]
Yeah, but it looks like you already did that commit?

**Comment by AVVS** [comment]
did I? oh, totally forgot about it :)

**Pull request closed without merging**